textdb/textdb-dataflow/src/test/java/edu/uci/ics/textdb/dataflow/source/SampleDataStore.java

```diff
@@ -0,0 +1,75 @@
+package edu.uci.ics.textdb.dataflow.source;
+
+import java.io.IOException;
+import java.nio.file.Paths;
+
+import org.apache.lucene.analysis.Analyzer;
+import org.apache.lucene.analysis.standard.StandardAnalyzer;
+import org.apache.lucene.document.Document;
+import org.apache.lucene.document.Field.Store;
+import org.apache.lucene.index.IndexWriter;
+import org.apache.lucene.index.IndexWriterConfig;
+import org.apache.lucene.index.IndexableField;
+import org.apache.lucene.store.Directory;
+import org.apache.lucene.store.FSDirectory;
+
+import edu.uci.ics.textdb.api.common.IField;
+import edu.uci.ics.textdb.api.common.ITuple;
+import edu.uci.ics.textdb.common.StringField;
+import edu.uci.ics.textdb.dataflow.common.SampleTuple;
+
+public class SampleDataStore {
+	public SampleDataStore(){
+		
+	}
+	
+	public void storeData() throws Exception{
+		IndexWriter indexWriter = null;

```

Comment:
why the indentation became 8? 
could you update your IDE settings to use space other than tab?